lib/spreadbase/codecs/open_document_12.rb

```diff
@@ -106,6 +106,15 @@ def decode_content_xml(content_xml_data, options={})
 
       private
 
+      def read_content_xml(zip_buffer)
+        io = StringIO.new(zip_buffer)
+        if Zip::VERSION >= '3.0.0'
```

Comment:
This is a semantic mistake :smile::

```
3.0.2 :001 > '10.0.0' > '3.0.0'
 => false 
```

In these cases, one can:

- either wrap both values in `Gem::Version.new`
- or use `Gem.loaded_specs['rubyzip'].version`, and wrap the second value in `Gem::Version.new`

Reply:
Thank you for your reviewing!
I've changed the version check logic according to your advice.